scripts/export_smplx_bodies.py

```diff
@@ -0,0 +1,746 @@
+                # As a workaround we trigger desired viewport update by setting color value.
+                node_shader.inputs[0].default_value = node_shader.inputs[
+                    0
+                ].default_value
+        else:
+            if node_texture is None:
+                node_texture = nodes.new(type="ShaderNodeTexImage")
+
+            if (texture == "UV_GRID") or (texture == "COLOR_GRID"):
+                if texture not in bpy.data.images:
+                    bpy.ops.image.new(name=texture, generated_type=texture)
+                image = bpy.data.images[texture]
+            else:
+                if texture not in bpy.data.images:
+                    texture_path = texture
+                    image = bpy.data.images.load(texture_path)
+                else:
+                    image = bpy.data.images[texture]
+
+            node_texture.image = image
+
+            if len(node_texture.outputs[0].links) == 0:
+                links.new(node_texture.outputs[0], node_shader.inputs[0])
+
+        # Switch viewport shading to Material Preview to show texture
+        if bpy.context.space_data and bpy.context.space_data.type == "VIEW_3D":
+            bpy.context.space_data.shading.type = "MATERIAL"
+
+
+def setup_bones():
+    bpy.types.Bone.xml_link_name = bpy.props.StringProperty(
+        name="URDF xml link name", default="unset"
+    )
+    bpy.types.Bone.body_segment_mass = bpy.props.FloatProperty(
+        name="URDF body segment mass", default=5.0
+    )
+    bpy.types.Bone.ixx = bpy.props.FloatProperty(
+        name="Inertia value XX", default=1.0
+    )
+    bpy.types.Bone.iyy = bpy.props.FloatProperty(
+        name="Inertia value YY", default=1.0
+    )
+    bpy.types.Bone.izz = bpy.props.FloatProperty(
+        name="Inertia value ZZ", default=1.0
+    )
+    bpy.types.Bone.ixy = bpy.props.FloatProperty(
+        name="Inertia value XY", default=0.0
+    )
+    bpy.types.Bone.ixz = bpy.props.FloatProperty(
+        name="Inertia value XZ", default=0.0
+    )
+    bpy.types.Bone.iyz = bpy.props.FloatProperty(
+        name="Inertia value YZ", default=0.0
+    )
+    bpy.types.EditBone.body_segment_mass = bpy.props.FloatProperty(
+        name="URDF body segment mass", default=5.0
+    )
+    bpy.types.EditBone.ixx = bpy.props.FloatProperty(
+        name="Inertia value XX", default=1.0
+    )
+    bpy.types.EditBone.iyy = bpy.props.FloatProperty(
+        name="Inertia value YY", default=1.0
+    )
+    bpy.types.EditBone.izz = bpy.props.FloatProperty(
+        name="Inertia value ZZ", default=1.0
+    )
+    bpy.types.EditBone.ixy = bpy.props.FloatProperty(
+        name="Inertia value XY", default=0.0
+    )
+    bpy.types.EditBone.ixz = bpy.props.FloatProperty(
+        name="Inertia value XZ", default=0.0
+    )
+    bpy.types.EditBone.iyz = bpy.props.FloatProperty(
+        name="Inertia value YZ", default=0.0
+    )
+
+
+def main():
```

Comment:
Some comment here about how to run this from CLI w/ blender and linking to the README would be helpful.